### ISSUE TITLE
Use Proxy to intercept and pass all method calls

### ISSUE THREAD

**Issue by tshemsedinov** [metarhia/metacom#13]
Proposed client syntax:
```js
const client = metacom.connect('tcp://domain.name:2000');
const app = client.application('applicationName');
app.methodName(par1, par2, par3, (err, data) => {
  console.dir({ err, data });
});
```

**Comment by tshemsedinov** [comment]
Yes, so better solution:
```js
const client = metacom.connect('tcp://domain.name:2000');
const api = client.api('applicationName', 'apiName', [apiVersion]);
api.methodName(par1, par2, par3, (err, data) => {
  console.dir({ err, data });
});
```
because each interface may have different versions (latest by default).

**Comment by tshemsedinov** [comment]
Closed in favour of new version 0.1.x implementing new specification
